www/js/app.js

```diff
@@ -646,40 +646,37 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
   
     /**
      * Display the list of articles with the given array of DirEntry
-     * @param {Array.<DirEntry>} dirEntryArray
-     * @param {Integer} maxArticles
+     * @param {Array} dirEntryArray The array of dirEntries returned from the binary search
      */
-    function populateListOfArticles(dirEntryArray, maxArticles) {       
+    function populateListOfArticles(dirEntryArray) {
         var articleListHeaderMessageDiv = $('#articleListHeaderMessage');
-        var nbDirEntry = 0;
         if (dirEntryArray) {
-            nbDirEntry = dirEntryArray.length;
+            var nbDirEntry = dirEntryArray.length;
         }
 
         var message;
-        if (maxArticles >= 0 && nbDirEntry >= maxArticles) {
-            message = maxArticles + " first articles below (refine your search).";
-        }
-        else {
-            message = nbDirEntry + " articles found.";
+        if (nbDirEntry >= MAX_SEARCH_RESULT_SIZE) {
+            message = 'First ' + MAX_SEARCH_RESULT_SIZE + ' articles below (refine your search).';
+        } else {
+            message = nbDirEntry + ' articles found.';
         }
         if (nbDirEntry === 0) {
-            message = "No articles found.";
+            message = 'No articles found.';
```

Comment:
The use of double and single quotes was completely mixed up in this function, so I've standardized on single because it makes the html easier to do.

Reply:
OK, no problem